main.yml

```diff
@@ -11,6 +11,18 @@ access-requests:
     paths:
       - /internal/access-requests
 
+acs:
+  title: Advanced Cluster Security
+  api:
+    version:
+      - v1
+  description: Frontend for Red Hat employees to manage ACS instances
+  deployment_repo: https://github.com/stackrox/acs-ui-build
```

Comment:
@ryelo Are there any drwbacks that the build repo is outside our org? AFAIK it shouldn't be a problem, just maintenance reasons.

Reply:
I am helping @sachaudh with the build repo. We will see if Jenkins picks up the changes. If not we will discuss an acceptable solution.

Reply:
I am just worried for jenkins to pick up the changes. But if jenkins will be fine we def can keep it where it is right now (no need to stall this any further)

Reply:
AFAIK the build repo has to be within the RedHatInsights github org. There shouldn't be any trouble moving the repo if it doesn't pick it up.

Reply:
No worries. I'm not an organization member so what should I do to create a repo under RedHatInsights?